content/realtime/delta.textile

```diff
@@ -0,0 +1,222 @@
+---
+title: Delta mode subscription
+section: realtime
+index: 32
+jump_to:
+  Help with:
+    - Overview#overview
+    - Examples#examples
+  Parameters:
+    - delta#delta
+---
+
+h2(#overview). Overview
+
+Often a channel carries messages that represent a series updates to a particular object or document and, as such, there is a significant degree of similarily between successive messages. Delta mode is a way for a client to subscribe to a channel so that message payloads sent over the wire contain only the difference (ie the delta) between the present message and the previous message on the channel. The client can then apply the delta to the previous message to obtain the full payload. Using delta mode can signficantly reduce the encoded size of each message in the case that message payloads change by differences that are small relative to the size of the value. This reduction in size can reduce bandwidth costs, reduce transit latencies, and enable greater message throughput on a connection.
```

Comment:
Maybe we could mention here that using the deltas would normally result in reducing the message size even if the subsequent messages are not mutations-of-an-object but random data. Deltas are compression in a way, i.e. any repeating blocks that the matching algorithm can find will be reduced to only a single repetition.